infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/CorfuReplicationClusterManagerBaseAdapter.java

```diff
@@ -0,0 +1,52 @@
+package org.corfudb.infrastructure.logreplication.infrastructure.plugins;
```

Comment:
is this class needed?  We already have a default implementation of the adapter.

Reply:
It has the basic implementation that other clusterManagers can share and also it gives an example of the basic implementation flow.